src/include/rbd/librbd.hpp

```diff
@@ -38,6 +39,7 @@ namespace librbd {
     uint64_t id;
     uint64_t size;
     std::string name;
+    utime_t stamp;
```

Comment:
This would break the ABI of the existing interface since you've increased the size of the structure. Clients compiled against an older librbd but linked against the newer ones would experience memory corruption.